### PR TITLE
storage: add kv range test

### PR DESCRIPTION
**Pull request by yichengq**

### CODE REVIEW

File: storage/kvstore.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ var (
 
 	ErrTnxIDMismatch = errors.New("storage: tnx id mismatch")
 	ErrCompacted     = errors.New("storage: required reversion has been compacted")
+	ErrUnhappenedRev = errors.New("storage: required reversion has not happened")
```
Comment (xiang90) [inline storage/kvstore.go]: ErrFutureRev?

Comment (yichengq) [inline storage/kvstore.go]: I don't think out a good error message for it.
is `required version is a future version` good?

Comment (xiang90) [inline storage/kvstore.go]: required reversion is a future reversion?

**Comment by xiang90** [comment]
LGTM

**Pull request merged**